scripts/uberenv/packages/umpire/package.py

```diff
@@ -80,7 +80,6 @@ class Umpire(CMakePackage, CudaPackage):
     variant('fortran', default=False, description='Build C/Fortran API')
     variant('c', default=True, description='Build C API')
     variant('numa', default=False, description='Enable NUMA support')
-    variant('cuda', default=False, description='Enable CUDA support')
```

Comment:
I assume this is because the `cuda` variant is inherited from `CudaPackage`?

Reply:
Yes

Reply:
Whoops, I didn't see your description :)

Reply:
We already discussed it, I made the mistake again :)